Makefile

```diff
@@ -87,7 +87,7 @@ lint: ## check style with flake8 and isort
 fix-lint: ## fix lint issues using autoflake, autopep8, and isort
 	find rdt tests -name '*.py' | xargs autoflake --in-place --remove-all-unused-imports --remove-unused-variables
 	autopep8 --in-place --recursive --aggressive rdt tests
-	isort --apply --atomic --recursive rdt tests
```

Comment:
why removing the `--recursive` ?

Reply:
It's [deprecated](https://pycqa.github.io/isort/docs/upgrade_guides/5.0.0.html).